tests/compile/test_silu_mul_quant_fusion.py

```diff
@@ -46,13 +54,46 @@ def forward(self, x):
         return x2
 
 
-@pytest.mark.parametrize("num_tokens", [256])
-@pytest.mark.parametrize("hidden_size", [64])
+class TestSiluMulNvfp4QuantModel(torch.nn.Module):
+
+    fused_op = torch.ops._C.silu_and_mul_nvfp4_quant.default
+    quant_op = torch.ops._C.scaled_fp4_quant.default
```

Comment:
Could this reference `FUSED_OPS` and `QUANT_OPS` instead?

Reply:
Also this could use `ops_in_model_before` (see other tests on how that's checked)